File: src/main/java/com/spring/dto/response/UserPresentation.java
package com.spring.dto.response;

import com.spring.constants.UserType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserPresentation {
    private Long id;
    private String avatar;
    private String firstName;
    private String lastName;
    private String email;
    private String gender;
    private String birthDay;
    private String phone;
    private Integer status;
    private Long createdBy;
    private Long lastModifiedBy;
    private String createdDate;
    private String lastModifiedDate;
    private UserType userType;
}

File: MSMA/app/src/main/java/vn/edu/usth/msma/data/dto/response/UserPresentation.kt
package vn.edu.usth.msma.data.dto.response

import vn.edu.usth.msma.utils.constants.UserType

data class UserPresentation(
    val id: Long,
    val avatar: String,
    val firstName: String,
    val lastName: String,
    val email: String,
    val gender: String,
    val birthDay: String,
    val phone: String,
    val status: Int,
    val createdBy: Long,
    val lastModifiedBy: Long,
    val createdDate: String,
    val lastModifiedDate: String,
    val userType: UserType
)